pkg/utils/version.go

```diff
@@ -97,3 +98,19 @@ func IsValidFor(actual string, minimum string) bool {
 	vb := ParseVersion(minimum)
 	return va.IsUndefined() || !va.LessRecentThan(vb)
 }
+
+func GetVersionFromImageTag(imageTag string) string {
```

Comment:
Very minor point, but strictly speaking I believe the argument passed in is an image descriptor comprised of a full image 'url' and a SHA. It is not a 'tag' in the usual sense of the word. A different name for the argument to this function might make it more obvious sometime in the future when someone looks at it in isolation.